lib/src/practice/quiz/widgets/quiz_app_bar.dart

```diff
@@ -0,0 +1,51 @@
+import "package:flutter/material.dart";
+import "package:flutter_gen/gen_l10n/app_localizations.dart";
+import "package:kana_to_kanji/src/core/widgets/rounded_linear_progress_indicator.dart";
+
+class QuizAppBar extends StatelessWidget implements PreferredSizeWidget {
+  final double progressBarValue;
+  final void Function() onClosePressed;
+  final Future<void> Function() onSkipPressed;
+
+  const QuizAppBar({
+    required this.progressBarValue,
+    required this.onClosePressed,
+    required this.onSkipPressed,
+    super.key,
+  });
+
+  @override
+  PreferredSizeWidget build(BuildContext context) {
+    final l10n = AppLocalizations.of(context);
+
+    return AppBar(
+      centerTitle: true,
+      backgroundColor: Colors.transparent,
+      leading: IconButton(
+        icon: const Icon(Icons.close),
+        onPressed: onClosePressed,
+      ),
+      title: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        children: [
+          SizedBox(
+            width: MediaQuery.of(context).size.width * 0.60,
+            child: RoundedLinearProgressIndicator(
+              value: progressBarValue,
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(left: 12.0),
+            child: IconButton(
+              icon: Text(l10n.quiz_skip_question),
+              onPressed: onSkipPressed,
+            ),
```

Comment:
This should be on the `AppBar.actions` section without the padding

Reply:
Updated